### PR TITLE
mens.tasclap.jp-2

### PR DESCRIPTION
**Pull request by SKEIDs**
moved from #103014

### CODE REVIEW

**Comment by Yuki2718** [comment]
Did you open this to avoid conflicts? Then no worry, we can solve conflicts.

**Comment by SKEIDs** [comment]
> Did you open this to avoid conflicts? If so, no need to worry about, we can solve conflicts.

I decided to rebuild the repository to deal with the Fork commit log coming up.
I figured it would not be good to lose the original pull request that has not been completed yet.
So I sent the pull request again.
Sorry for,I don't know how to use github.

**Comment by Yuki2718** [comment]
> Sorry for,I don't know how to use github.

No probelm, I didn't too at first.

**Comment by SKEIDs** [comment]
> No probelm, I didn't too at first.

Is it possible to merge even if the original fork repository is gone?

**Comment by Yuki2718** [comment]
Yes, possible so you don't need to do.

**Comment by SKEIDs** [comment]
> Yes, possible so you don't need to do.
oh,thank you so much

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -46,6 +46,7 @@ originalnews.nico,archives.gov,sportnews.to,nationalchronicle.in,analogictips.co
 ! ##.sns_area
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
+mens.tasclap.jp##.p-article-main-bottom-share
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:49]: Better `mens.tasclap.jp##.p-article-main-bottom-share::before`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt]: ok,thank you for advice

**Pull request merged**